haystack/components/generators/chat/hugging_face_api.py

```diff
@@ -384,6 +387,34 @@ def _run_streaming(
 
         return {"replies": [message]}
 
+    @staticmethod
+    def _convert_hfapi_tool_call(hfapi_tc) -> Optional[ToolCall]:
```

Comment:
```suggestion
    def _convert_hfapi_tool_call(hfapi_tc: "ChatCompletionOutputToolCall") -> Optional[ToolCall]:
```

let's add the type hint also import this object in the `LazyImport` block